src/vs/workbench/browser/actions/workspaceCommands.ts

```diff
@@ -19,6 +19,8 @@ import { getIconClasses } from 'vs/editor/common/services/getIconClasses';
 import { IModelService } from 'vs/editor/common/services/modelService';
 import { IModeService } from 'vs/editor/common/services/modeService';
 import { IFileDialogService } from 'vs/platform/dialogs/common/dialogs';
+import { IWindowService } from 'vs/platform/windows/common/windows';
```

Comment:
Please revert changes to this file now they're no longer needed

Reply:
Oops, missed this. Will do.